src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/CachedBitmap.Windows.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Drawing.Imaging
+{
+    public sealed partial class CachedBitmap
+    {
+        internal static bool IsCachedBitmapSupported()
+        {
+            // On Windows, CachedBitmap is always supported.
+            return true;
+        }
```

Comment:
Make this a single line?

```suggestion
        internal static bool IsCachedBitmapSupported() => true;
```